utils/nano/Makefile

```diff
@@ -63,7 +69,36 @@ define Package/nano-plus/description
   $(call Package/nano/description)
 endef
 
-ifeq ($(BUILD_VARIANT),plus)
+define Package/nano-full/description
+  nano-full - most features enabled, including syntax highlighting,
+  multibuffer, Unicode/UTF-8, help, justify, nanorc, some key bindings)
+
+  $(call Package/nano/description)
+endef
+
+ifeq ($(BUILD_VARIANT),full)
+# full variant with most features included
+  CONFIGURE_ARGS += \
+	--enable-help \
+	--enable-justify \
+	--enable-linenumbers \
+	--enable-multibuffer \
+	--enable-nanorc \
+	--enable-utf8 \
+	--enable-browser \
+	--enable-color \
+	--enable-comment \
+	--disable-extra \
+	--disable-histories \
+	--enable-justify \
```

Comment:
At least justify and linenumbers are defined twice...

Reply:
Oops..

Reply:
duplicate configure arguments removed.

Reply:
@hnyman ping